### PR TITLE
Analytics documentation rewrite

### PR DESCRIPTION
**Pull request by cvkef**

### CODE REVIEW

File: source/localizable/analytics/index.html.md.erb [hunk 1]
```diff
@@ -26,56 +32,50 @@ section.
...
+#### Complete Example
 
-* Configuration
-  * <%= link_to 'Analytics Settings', '/analytics/settings' %>
-  * <%= link_to 'Ecommerce metrics', '/analytics/ecommerce' %>
-* <%= link_to t('docs.analytics.examples'), '/analytics/examples' %>
+<% tracking_script = YAML.load_file(File.expand_path(File.join('./resources',
+                                                               'code_examples',
+                                                               "analytics/#{flavor}/js_snippet.yml"))) %>
+<% complete_example = YAML.load_file(File.expand_path(File.join('./resources',
```
Comment (Zorbash) [inline source/localizable/analytics/index.html.md.erb]: Why assign a local variable here?

**Comment by fotos** [comment]
+1 LGTM

File: source/localizable/analytics/index.html.md.erb (hunk 1 above)
Comment (fotos) [inline source/localizable/analytics/index.html.md.erb]: We already did that with the tracking script. We just extracted the `complete_example` as well.

**Comment by Zorbash** [comment]
+1 LGTM beware the :bear:

**Pull request merged**